testsuite/run_sets/testsuite.yml

```diff
@@ -86,7 +84,7 @@
 - features/min_docker_auth_registry.feature
 - features/srv_docker_advanced_content_management.feature
 - features/srv_docker_cve_audit.feature
-# - features/min_osimage_build_image.feature
+- features/min_osimage_build_image.feature
```

Comment:
Not sure about this one. The problem was, that it crash the testsuite. With it enabled we got very often no result.
If this is still the case we need to keep it disabled until somebody could debug and fix it.

Reply:
OK, removing.